### DISCUSSION TITLE
🚀 Troubleshooting Guide: Share Your Tips

### DISCUSSION THREAD

**Discussion by rabwill** [microsoft/copilot-camp#332] [General]
Got any tips for solving common setup issues? Drop your wisdom and help the community thrive!

**Comment by swatiarora11** [comment]
I am able to provision the north wind messaging extension typescript sample (downloaded from Copilot for M365 github repo) in the demo tenant. All functionalities work fine with copilot in teams canvas. While provisioning the python implementation of the same sample using visual studio code teams toolkit, I am getting "We could not find a subscription" error. Can someone help me on this issue?
![image](https://github.com/user-attachments/assets/bd53ac6c-2f57-46e5-b0ed-f1073d65f71b)

**Comment by rhshanbhag** [comment]
Ensure you have Node JS Version 18.16.0. For easy Node JS version management , Install NVM (Node Version Manager) and Use specific version of Node JS before provisioning the Lab Exercises.

nvm install 18.16.0
 Once the installation is complete  execute below command
 nvm use 18.16.0
